### DISCUSSION TITLE
Flare RT and Varia 515 Taillights

### DISCUSSION THREAD

**Discussion by michalek-marcin** [maca88/SmartBikeLights#50] [General]
Hi,

I installed the app on garmin edge 840. Everything works, except one thing. 
Is it possible to connect flare rt and varia taillights and configure them using this app? I would like to disable varia light, keep radar on and control flare rt light mode.
When I connect varia using light network settings (via garmin settings) I receive error 3 code on Smart bike lights data screen. When I remove it from the lights and keep it only as a radar everything is ok.

**Comment by maca88** [comment]
Hi,

If you want to just control flare RT with the Smart Bike Lights then you need to specify the serial number when creating the configuration in the Lights Configurator. If you would like to control both lights then you will have to install also "Smart Bike Lights #2" data field and setup one light per data field. You can also check the documentation for controlling multiple lights of the same type: https://github.com/maca88/SmartBikeLights#control-multiple-lights-of-the-same-type

**Comment by vanillea6** [comment]
Hi,

I want to use an Ion 200 RT lamp at the front and a Varia 515 at the back, alternating with a Flare RT lamp. Varia 515 is entered in Smart Bike Lights and Flare RT in SBL#2. But I still get error message 3. The same with the serial number. I don't understand why SBL also takes a lamp into account that hasn't been programmed at all. Or does SBL interact with the lighting network that is pre-installed? Thank you very much!
